frontend/pages/group/mealplan/planner/edit.vue

```diff
@@ -300,19 +311,42 @@ export default defineComponent({
       text: "",
       recipeId: undefined as string | undefined,
       entryType: "dinner" as PlanEntryType,
+      existing: false,
+      id: 0,
+      groupId: ""
```

Comment:
Can you explain the usecase for `groupId` please? (in general, as it doesn't exist in the program prior to your changes)

Reply:
Both id and groupId are unneeded here, was making sure there weren't any type errors but works and lints fine without them.